src/main/java/com/trueffect/logic/DateOperation.java

```diff
@@ -0,0 +1,155 @@
+package com.trueffect.logic;
+
+import com.trueffect.tools.ConstantData;
+import com.trueffect.tools.RegularExpression;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.regex.Pattern;
+import org.joda.time.LocalDate;
+import org.joda.time.Period;
+
+/**
+ *
+ * @author santiago.mamani
+ */
+public class DateOperation {
+
+    public static boolean isValidDateFormat(String date) {
+        return Pattern.matches(RegularExpression.DATE, date);
```

Comment:
Class DateOperation

dateIs-InRangeValid : objective looks like validate a range of dates

diferenceYear : what will be happend when  catch is executed
catch (ParseException ex) 
isLess :what will be happend when  catch is executed
areSameMonthAndYear  :what will be happend when  catch is executed
diferenceMinutes:what will be happend when  catch is executed
 diferenceDays:what will be happend when  catch is executed